component_layout_plugin.py

```diff
@@ -157,10 +157,8 @@ def Run( self ):
                 else:
                     mod.SetPosition(pcbnew.wxPointMM(x0 + x, y0 + y))
             
-            if flip and not mod.IsFlipped():
-                mod.Flip(mod.GetPosition())
-            if not flip and mod.IsFlipped():
-                mod.Flip(mod.GetPosition())
+            if flip ^ (mod.IsFlipped()):
+                mod.Flip(mod.GetPosition(), False)
```

Comment:
```suggestion
                if v5_compat:
                    mod.Flip(mod.GetPosition())
                else:
                    mod.Flip(mod.GetPosition(), False)
```

The aFlipLeftRight argument is new in v6.